src/modules/powerrename/PowerRenameUILib/MainWindow.xaml

```diff
@@ -279,9 +279,9 @@
                         <x:String>Extension only</x:String>
                     </ComboBox>
                     <AppBarSeparator Margin="5,0,5,0" />
-                    <ToggleButton x:Name="toggleButton_includeFiles" Content="&#xE160;" MinHeight="0" Height="31" IsChecked="True" FontFamily="{ThemeResource SymbolThemeFontFamily}" x:Uid="ToggleButton_IncludeFiles" Style="{StaticResource CustomToggleButtonStyle}" />
+                    <ToggleButton x:Name="toggleButton_includeFiles" Content="&#xE7C3;" MinHeight="0" Height="31" IsChecked="True" FontFamily="{ThemeResource SymbolThemeFontFamily}" x:Uid="ToggleButton_IncludeFiles" Style="{StaticResource CustomToggleButtonStyle}" />
                     <ToggleButton x:Name="toggleButton_includeFolders" Content="&#xE8B7;" MinHeight="0" Height="31" IsChecked="True" FontFamily="{ThemeResource SymbolThemeFontFamily}" x:Uid="ToggleButton_IncludeFolders" Style="{StaticResource CustomToggleButtonStyle}" />
-                    <ToggleButton x:Name="toggleButton_includeSubfolders" Content="&#xE12F;" MinHeight="0" Height="31" IsChecked="True" FontFamily="{ThemeResource SymbolThemeFontFamily}" x:Uid="ToggleButton_IncludeSubFolders" Style="{StaticResource CustomToggleButtonStyle}" />
+                    <ToggleButton x:Name="toggleButton_includeSubfolders" Content="&#xED41;" MinHeight="0" Height="31" IsChecked="True" FontFamily="{ThemeResource SymbolThemeFontFamily}" x:Uid="ToggleButton_IncludeSubFolders" Style="{StaticResource CustomToggleButtonStyle}" />
```

Comment:
This looks like vertical folder? Not sure if it's better than the original folder, which atleast showed some sort of nesting/subfolders?

Reply:
E12F is not only deprecated, but also a right-aligned bulleted list? We've had a discusson about using check boxes earlier and I still stand behind that.